File: acm_statistic_backend/api/api/DTO/performance/aeroport/RedevanceAeroportTypeDTO.cs
namespace api.DTO.performance.aeroport
{
    public class RedevanceAeroportTypeDTO
    {
        public string Type_trafic { get; set; }
        public decimal Montant { get; set; }
        public string Aeroport { get; set; }
        public int Passagers { get; set; }
        public int Vols { get; set; }
    }
}

File: acm_statistic_frontend/Interface Tableau de Bord Analytique Aérien/src/models/performanceAeroport/RedevanceAeroportTypeDTO.ts
export class RedevanceAeroportTypeDTO{
    passagers: number;
    vols :number; 
    montant: number;
    aeroport: string | null;
    type_trafic: string | null;

    constructor(init?: Partial<RedevanceAeroportTypeDTO>) {
    Object.assign(this, init);
    }   
}